MoreCombineVoiceLines.lua

```diff
@@ -0,0 +1,615 @@
+Schema.voices.Add("combine", "STABILZATION TEAM HOLDING AT POSTION", "Stabilzation team holding at position", "npc/combine_soldier/vo/stabilizationteamholding.wav")
+Schema.voices.Add("combine", "STANDING BY", "Standing by", "npc/combine_soldier/vo/standingby].wav")
+Schema.voices.Add("combine", "STAR", "Star", "npc/combine_soldier/vo/star.wav")
+Schema.voices.Add("combine", "STAY ALERT", "Stay alert", "npc/combine_soldier/vo/stayalert.wav")
+Schema.voices.Add("combine", "STAY ALERT REPORT SIGHT LINES", "Stay alert report sight lines" ,"npc/combine_soldier/vo/stayalertreportsightlines.wav")
+Schema.voices.Add("combine", "STINGER", "Stinger", "npc/combine_soldier/vo/stinger.wav")
+Schema.voices.Add("combine", "STORM", "Storm", "npc/combine_soldier/vo/storm.wav")
+Schema.voices.Add("combine", "STRIKER", "Striker", "npc/combine_soldier/vo/striker.wav")
+Schema.voices.Add("combine", "SUNDOWN", "Sundown", "npc/combine_soldier/vo/sundown.wav")
+Schema.voices.Add("combine", "SUPPRESSING", "Suppressing", "npc/combine_soldier/vo/suppressing.wav")
+Schema.voices.Add("combine", "SWARM OUT BREAK IN SECTOR", "Swarm out break in sector", "npc/combine_soldier/vo/swarmoutbreakinsector.wav")
+Schema.voices.Add("combine", "SWEEPER", "Sweeper", "npc/combine_soldier/vo/sweeper.wav")
+Schema.voices.Add("combine", "SWEEPING IN", "Sweeping in", "npc/combine_soldier/vo/sweepingin.wav")
+Schema.voices.Add("combine", "SWIFT", "Swift", "npc/combine_soldier/vo/swift.wav")
+Schema.voices.Add("combine", "SWORD", "Sword", "npc/combine_soldier/vo/sword.wav")
+Schema.voices.Add("combine", "TARGET", "Target", "npc/combine_soldier/vo/target.wav")
+Schema.voices.Add("combine", "TARGET COMPROMISED", "Target compromised move in, move in", "npc/combine_soldier/vo/targetcompromisedmovein.wav")
+Schema.voices.Add("combine", "WARP UP", "Thats it warp up", "npc/combine_soldier/vo/thatsitwrapitup.wav")
+Schema.voices.Add("combine", "UNIFORM", "Uniform", "npc/combine_soldier/vo/thatsitwrapitup.wav")
+Schema.voices.Add("combine", "UNIT IS CLOSING", "Unit is closing", "npc/combine_soldier/vo/unitisclosing.wav")
+Schema.voices.Add("combine", "UNIT IS INBOUND", "Unit is inbound", "npc/combine_soldier/vo/unitisinbound.wav")
+Schema.voices.Add("combine", "VAMP", "Vamp", "npc/combine_soldier/vo/vamp.wav")
+Schema.voices.Add("combine", "VISCON", "Viscon", "npc/combine_soldier/vo/viscon.wav")
+Schema.voices.Add("combine", "VISUAL ON EXOGENS", "Visual on exogens", "npc/combine_soldier/vo/visualonexogens.wav")
+Schema.voices.Add("combine", "WEAPONS OFF SAFE", "Weapons off safe prep for contact")
+Schema.voices.Add("combine", "WE HAVE FREE PARASITES", "We have free parasites", "npc/combine_soldier/vo/wehavefreeparasites.wav")
+Schema.voices.Add("combine", "WINDER", "Winder", "npc/combine_soldier/vo/winder.wav")
+
+Schema.voices.AddClass("combine", function(client)

```

Comment:
The HL2 RP schema already comes with a `Combine` voice class, making this redundant

Reply:
Changed it to `comb`